tests/saitests/py3/sai_qos_tests.py

```diff
@@ -3746,6 +3747,19 @@ def runTest(self):
 
         send_packet(self, src_port_id, pkt, pkts_num_leak_out)
 
+        if 'hwsku' in self.test_params and self.test_params['hwsku'] in ('Arista-7060X6-64PE-256x200G'):
+            pkt = construct_ip_pkt(default_packet_length,
+                                   pkt_dst_mac,
+                                   src_port_mac,
+                                   src_port_ip,
+                                   dst_port_ip,
+                                   9,
+                                   src_port_vlan,
+                                   ip_id=exp_ip_id,
+                                   ecn=ecn,
+                                   ttl=64)
+            send_packet(self, src_port_id, pkt, int(pkts_num_egr_mem * 64 / default_packet_length))
```

Comment:
once we have the packet length changed, we can use the pkts_num_egr_mem directly without any magic calculations.

Reply:
238*64/1500 == 10.15, which seem to be smaller than the ones we need.